src/java/Main.java

```diff
@@ -0,0 +1,114 @@
+import com.entities.Board;
+import com.entities.Ladder;
+import com.entities.Player;
+import com.entities.Snake;
+
+import java.util.List;
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args){
+        int numberOfSnakes = Integer.parseInt(args[0]);
+        int numberOfLadders = Integer.parseInt(args[numberOfSnakes + 1]);
+        int numberOfPlayers = Integer.parseInt(args[numberOfSnakes + numberOfLadders + 2]);
+
+        Board board = new Board();
+        for (int i = 1; i <= numberOfSnakes; i++) {
+            String[] snakePositions = args[i].split(" ");
+            int head = Integer.parseInt(snakePositions[0]);
+            int tail = Integer.parseInt(snakePositions[1]);
+            board.addSnake(head, tail);
+        }
+        for (int i = numberOfSnakes + 2; i <= numberOfSnakes + numberOfLadders + 1; i++) {
+            String[] ladderPositions = args[i].split(" ");
+            int start = Integer.parseInt(ladderPositions[0]);
+            int end = Integer.parseInt(ladderPositions[1]);
+            board.addLadder(start, end);
+        }
+
+        for (int i = numberOfSnakes + numberOfLadders + 3; i < args.length; i++) {
+            board.addPlayer(args[i]);
+        }
+
+        playGame(board);
+    }
+
```

Comment:
you could have created a game class and added playgame() in it.
Advice is to keep main() as minimal as possible